src/App.tsx

```diff
@@ -27,7 +27,7 @@ export default function App() {
     const currentId = id.get();
     const newPerson = {
       id: currentId,
-      name: `Person ${getRandomInt(1000)}`,
+      name: `Person ${currentId}`,
```

Comment:
Have you tested this with a deleted person followed by an added person followed by another deleted person?

Reply:
Also I would prefer if this part was in a separate PR